src/server/migrations/20201129173036_create_feedback_table.js

```diff
@@ -0,0 +1,22 @@
+exports.up = function (knex) {
```

Comment:
This table should be deleted also in the remote. You only deleted locally but not in the remote.